GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -85,20 +88,31 @@ public void EnsureAuthenticated()
                     uri = new Uri(Url);
                 }
 
-                // TODO: Use TfsTeamProjectCollection constructor that takes a TfsClientCredentials object
+#if VS2010
```

Comment:
I'd rather not introduce `#if`. Other version-specific stuff uses subclasses or composition to vary the behavior. Can that be done with these changes, too?